src/handlers/processors.ts

```diff
@@ -79,7 +78,7 @@ export const processors: Record<string, Handler> = {
     post: [],
   },
   [GitHubEvent.PUSH_EVENT]: {
-    pre: [validateConfigChange],
+    pre: [],
```

Comment:
Why is this removed?

Reply:
Do you have another approach for reimplementation of this feature? The purpose of it as you may recall is to immediately diagnose broken configs. Which can happen regularly with partners as they update them.

Reply:
I just moved it [here](https://github.com/ubiquity-whilefoo/ubiquibot/blob/110088837ccb21d408781076e882514e4fd8d3ac/src/bindings/event.ts#L60), it's still working.
Now thinking about it, it might make sense to merge it with `loadConfiguration`